catalog/app/containers/AddToPackage/Provider.tsx

```diff
@@ -0,0 +1,34 @@
+import { basename } from 'path'
+
+import * as R from 'ramda'
+import * as React from 'react'
+
+import type * as Model from 'model'
+
+const Ctx = React.createContext<{
+  append: (file: Model.S3File) => void
+  entries: Record<string, Model.S3File>
+}>({
+  append: () => {},
```

Comment:
you probably don't want this dummy context, bc it will just silently swallow data when there's no provider in tree. maybe just throw an error (e.g. 'Please initialize the provider') in `append()` fn

Reply:
You're right, thanks

Reply:
I'll add in next PR, where I turn this feature on